submissions/balancer-design/universal_8to8.md

```diff
@@ -0,0 +1,88 @@
+## Universal 8-8: Perfectly balanced, as all things should be
+
+Some time ago I made a bunch of computer parts with belts and splitters. This time I will continue my unhealthy obsession with moving items around by making something that's actually at least slightly practical, that being a balancer. Now, why would I write a whole article on designing a balancer? Well that is because the one we will be making today is no average balancer. This article won't go very deep into balancer design principles, since we will be using pre-made and well-known blueprints for the most part. What you *do* need to know is some terminology:
```

Comment:
> What you do need to know are the various properties a balancer can have:

You could also first mention that all balancers aren't equal. I think it's a common misconception that balancers just work, no matter what application or shape. So then say, "all aren't equal", "there are these different types". Don't want to lose inexperienced players in the first paragraph :P